airflow/providers/amazon/aws/example_dags/example_sagemaker.py

```diff
@@ -0,0 +1,179 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from datetime import datetime
+from os import environ
+
+from airflow import DAG
+from airflow.providers.amazon.aws.operators.sagemaker import (
+    SageMakerDeleteModelOperator,
+    SageMakerModelOperator,
+    SageMakerProcessingOperator,
+    SageMakerTrainingOperator,
+    SageMakerTransformOperator,
+)
+
+model_name = "sample_model"
+training_job_name = 'sample_training'
```

Comment:
Here and elsewhere:  Single quotes or double quotes, please pick one.  Also, all of these variables defined before the DAG context are effectively globals/constants and names should be in all caps, such as `MODEL_NAME`.

Reply:
Have corrected the quotes and global variables. Thanks!

Reply:
I appreciate that thanks.